test/distributed/_shard/sharded_tensor/test_sharded_tensor.py

```diff
@@ -918,7 +919,7 @@ def test_invalid_pg_rpc_ranks(self):
         self.init_pg()
 
         # Init RPC with different ranks.
-        rpc_backend_options = rpc.TensorPipeRpcBackendOptions()
+        rpc_backend_options = rpc.TensorPipeRpcBackendOptions(_transports=tp_transports())
```

Comment:
Do the EFA workarounds have anything to do with `ShardedTensor.gather`? If not, can we move these changes to a separate PR?

Reply:
Sure, it's here because I have a hard time working on it on AWS otherwise.

Reply:
Looks like we still have RPC changes here, shouldn't this go in https://github.com/pytorch/pytorch/pull/77363?